### PR TITLE
feat: code editor line numbers

### PR DESCRIPTION
**Pull request by lottamus**
Been wanting to do this for a while...

**Code Viewer Before**

image

**Code Viewer After**

image

**Code Editor**

image

### CODE REVIEW

**Comment by marbemac** [comment]
I can't tell a difference in the images haha. What am I looking at?

**Comment by lottamus** [comment]
@marbemac I added support for line numbers in the code editor which we didn’t have before. 

Took screenshots of the before and after of the code viewer to show no changes and one of the code editor to show it matches

File: src/CodeViewer/index.tsx [hunk 1]
```diff
@@ -44,18 +43,16 @@ const CodeViewer: React.FunctionComponent<ICodeViewerProps> = ({
     );
   }
 
-  const markup = parseCode(value, lang, showLineNumbers);
+  const code = highlightCode(value, lang, showLineNumbers);
```
Comment (marbemac) [inline src/CodeViewer/index.tsx]: Why did we have parseCode before? What's changing here?

File: src/CodeViewer/index.tsx [hunk 2]
```diff
@@ -44,18 +43,16 @@ const CodeViewer: React.FunctionComponent<ICodeViewerProps> = ({
...
     <pre
       className={cn(Classes.CODE_EDITOR, className, `language-${lang || 'unknown'}`, {
         isInline: inline,
-        showLineNumbers,
       })}
       {...(rest as React.HTMLAttributes<HTMLPreElement>)}
-    >
-      {markup ? markup.map(astToReact()) : value}
-    </pre>
+      {...{ dangerouslySetInnerHTML: { __html: code } }}
```
Comment (marbemac) [inline src/CodeViewer/index.tsx]: Can probably just do `dangerouslySetInnerHTML={{ __html: code }}`?

Comment (lottamus) [inline src/CodeViewer/index.tsx]:
```suggestion
      dangerouslySetInnerHTML={{ __html: code }}
```

**Comment by wmhilton** [comment]
Merge it before the p̷a̷r̷s̷i̵n̴g̶ ̷ d̸͋̐̃͂̈e̷̎̌̏̔̏m̴̓͆̊̚͝ő̷̀̒͐̅n̵̊̊̊́͘s̴̔̔̔̀͘ g̰͓̹̜̮̏̆̿̅̕e͖̠̞̻̩͆̆̾͝͝ţ̥̲̬̪̊̐͊̀͆ ̗͕͓̖̗͗́̎͝͝ư̙̠̮̹͐̒͒͝ͅs̡͇̞͎̠̅͆̈́͘͝

**Pull request merged**